### DISCUSSION TITLE
Help requested: datetime_precision error on mysql introspect.

### DISCUSSION THREAD

**Discussion by SirCameron** [prisma/prisma#4944] [Q&A]
Hi

I'm getting an error when introspecting a mysql database:
```Error: Error in connector: Error querying the database: Error querying the database: Error accessing result set, column not found: datetime_precision```

I've googled, but unable to find anything helpful..

mysql version is:
```mysql  Ver 14.14 Distrib 5.5.60```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
